packages/compiler/src/server/yaml-resolver.ts

```diff
@@ -85,7 +102,11 @@ export function resolveYamlScalarTarget(
         path: [""],
         type: "key",
         source: "",
+        sourceType: "PLAIN",
         siblings: rootProperties,
+        yamlDoc,
+        nodePostionRange: { pos: 0, end: 0 },
```

Comment:
it's better to return undefined to tell the caller not range is provided. [0, 0] is still a valid range

Reply:
same comment for below